### PR TITLE
Arm 7.17.5

### PR DESCRIPTION
**Pull request by anupteal**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Title of the PR starts with installer/product name (e.g. `[ansible/artifactory]`)
- [ ] CHANGELOG.md updated
- [ ] Variables and other changes are documented in the README.md

<!--
Thank you for contributing . 

Latest changes has committed for Artifactory version 7.17.5 & X-ray version 3.22.1 in all required files.
@danielmkn , Please verify and merge it.

### CODE REVIEW

File: AzureResourceManager/Artifactory/azuredeploy.json [hunk 1]
```diff
@@ -297,7 +297,7 @@
     "artifactoryLicense5": "[parameters('artifactoryLicense5')]",
     "osType": {
       "publisher": "jfrog",
-      "offer": "artifactory-vm",
+      "offer": "artifactory-vm-preview",
```
Comment (danielmkn) [inline AzureResourceManager/Artifactory/azuredeploy.json:301]: Please remove "-preview"

File: AzureResourceManager/Artifactory/azuredeploy.json [hunk 2]
```diff
@@ -765,7 +765,7 @@
       "plan": {
         "name": "artifactory-vm",
         "publisher": "jfrog",
-        "product": "artifactory-vm"
+        "product": "artifactory-vm-preview"
```
Comment (danielmkn) [inline AzureResourceManager/Artifactory/azuredeploy.json:769]: Please remove "-preview"

File: AzureResourceManager/Artifactory/azuredeploy.json [hunk 3]
```diff
@@ -862,7 +862,7 @@
...
```
Comment (danielmkn) [inline AzureResourceManager/Artifactory/azuredeploy.json:866]: Please remove "-preview"

**Review by danielmkn — CHANGES_REQUESTED** [review]

**Comment by anupteal** [comment]
-preview removed from all three places. Please review it again and merge it.

**Review by danielmkn — APPROVED** [review]

**Pull request merged**